packages/kbn-optimizer/limits.yml

```diff
@@ -1,110 +1,110 @@
 pageLoadAssetSize:
-  advancedSettings: 27596
```

Comment:
I don't think we should reset the limits file like this, we should let teams manage their limits and several teams have a larger budget that is still less than 100k which I think we should leave.